cabal.project

```diff
@@ -37,8 +37,9 @@ extra-packages: ieee, filemanip
 -- when decoding invalid (e.g. malicious) text literals.
 source-repository-package
   type: git
-  location: https://github.com/Quid2/flat.git
-  tag: ee59880f47ab835dbd73bea0847dab7869fc20d8
+  location: https://github.com/Quid2/flat
+  tag: 44dd79ab3d6547afd7674876f0307995fc40dfcf
+  -- 1q7l1wip28fnrrqlmns1v2csis4q2m0yzg8ivvy60935gg8ka9dz
```

Comment:
Please read CONTRIBUTING, which tells you what to do here. We don't use the comments in the cabal.project file, we put them in the `sha256map` in Nix code.